apps/web/pages/event-types/[type].tsx

```diff
@@ -346,40 +346,43 @@ const EventTypePage = (props: inferSSRProps<typeof getServerSideProps>) => {
   };
 
   useEffect(() => {
```

Comment:
You can extract this logic into a separate component, then enable like:

```javascript
{props.hasweb3Integration && <Web3TokenList />
```

Reply:
I just realized `tokensList` isn't even being used anywhere in the event type page. Do we even need to keep all those calls being made?